infrastructure/config/logging.template.yml

```diff
@@ -15,6 +15,7 @@ Resources:
             Resource:
               - !Sub arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/onboarding-self-service/* # Application prefix for audit log groups
               - !Sub arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/onboarding-product-pages/* # Application prefix for audit log groups
+              - !Sub arn:aws:logs:${AWS::Region}:${AWS::AccountId}:log-group:/onboarding-infrastructure/* # Application prefix for audit log groups
```

Comment:
could you please fix in the other repo; we are moving `config` folder to 
https://github.com/govuk-one-login/onboarding-self-service-infra